### PR TITLE
Added more Toit language code examples

### PR DESCRIPTION
**Pull request by HelenaMeyer**

### CODE REVIEW

File: language/fib.toit [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  n1 := 1
+  n2 := 1
+  i := 3
+
+  while i < n:
+    tmp := n2
+    n2 = n2 + n1
+    n1 = tmp
+    i = i + 1
```
Comment (floitsch) [inline language/fib.toit:34]: Why not use a `for` loop?
or at the very least `i++`.

Copied from Python?

File: language/mandelbrot.toit [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/**
+Mandelbrot example in Toit.
+
+This example draws a shape as the output.
+
```
Comment (floitsch) [inline language/mandelbrot.toit:5]: Remove this line.

File: language/mandelbrot.toit [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+MAX_PIXEL_VALUE ::= 255
+
+main:
+  mandelbrot 200 80 -0.6 0.0 0.015 500
+  mandelbrot 200 80 -0.743030 0.126433 0.00003 5000
+
+
+mandelbrot width height x_center y_center scale limit:
+  pixels := List width:
+    ByteArray height
```
Comment (floitsch) [inline language/mandelbrot.toit:17]: I would put it on the previous line, but ok like this.

Comment (HelenaMeyer) [reply language/mandelbrot.toit:17]: For readability, I will put it on the previous line.

**Pull request merged**